components/PluginCard.tsx

```diff
@@ -0,0 +1,40 @@
+import { Button, Card, CardActions, CardContent, Link, Typography } from "@mui/material"
+import React from 'react';
+
+interface PluginCardProps {
+    title: string
+    description: string
+    githubLink: string
+}
+
+const PluginCard: React.FC<PluginCardProps> = ({ title, description, githubLink }) => (
+  <Card
+    sx={{
+      height: '15rem',
+      display: 'flex',
+      flexDirection: 'column',
```

Comment:
flex properties here + allowing the content to grow means that buttons are pushed to the bottom of the card

Reply:
Oh that's cool, it looks better that way.